src/main/java/org/xwiki/git/GitManager.java

```diff
@@ -68,6 +68,25 @@ default Repository getRepository(String repositoryURI, String localDirectoryName
         throw new UnsupportedOperationException();
     }
 
+    /**
+     * Clone a private Git repository as Bare using the credentials provided by user and store it locally in the
```

Comment:
I don't think this is true since it should be possible to clone as bare even with no credentials. 

I think we're also missing a method for that BTW.

Reply:
Changed.